### DISCUSSION TITLE
谷歌网站评测的分数好像降低了欸

### DISCUSSION THREAD

**Discussion by fengguowudi** [hugo-fixit/FixIt#315] [General]
https://pagespeed.web.dev/analysis/https-fixit-lruihao-cn/wsm28n95xh?form_factor=mobile
https://pagespeed.web.dev/analysis/https-fixit-lruihao-cn/wsm28n95xh?form_factor=desktop

**Comment by Lruihao** [comment]
有时间再针对性改改
